README.md

```diff
@@ -0,0 +1,8 @@
+# tsgg
+sgg chat in the terminal
+
+`cp sample-config.json config.json`
+`go build`
+
+#### credits to original project
+https://github.com/voloshink/tdgg
```

Comment:
The original project kindly moved the repository over here, so we *are* the "original" now. Commits (and authorship) is preserved, so this can be removed.